### PR TITLE
Update documentation for create_db.sh

### PR DESCRIPTION
**Pull request by TobiTenno**
### Pull Request Checklist

* [x] I have added any new tests that need to pass to `sytest-whitelist` as specified in [docs/sytest.md](https://github.com/matrix-org/dendrite/blob/master/docs/sytest.md)
* [x] Pull request includes a [sign off](https://github.com/matrix-org/dendrite/blob/master/docs/CONTRIBUTING.md#sign-off)

#### Comments

discovered by #1687 

Signed-off-by: `Matt Voboril <tobiah@pm.me>`

### CODE REVIEW

File: build/docker/README.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ runtime config should come from. The mounted folder must contain:
 - `server.crt` certificate file
 - `server.key` private key file for the above certificate
 
+as well as 
+
+- `build_db.sh` in a parallel `postgres` parallel to the `config` folder, providing database generation.
```
Comment (Kegsay) [inline build/docker/README.md:39]: Don't you mean `create_db.sh`?

Comment (TobiTenno) [reply build/docker/README.md]: yes indeed.

**Comment by TobiTenno** [comment]
I have literally no code changes, so any code scanning problems are upstream...

**Comment by TobiTenno** [comment]
guess this isn't gonna happen.

**Pull request closed without merging**